gramps/gen/plug/report/utils.py

```diff
@@ -46,6 +47,8 @@
 from ...utils.file import media_path_full
 from ..docgen import IndexMark, INDEX_TYPE_ALP
 
+from gramps.gen.utils.symbols import Symbols
```

Comment:
This import should be made relative and moved next to the `media_path_full` import.